src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/InitializeHandler.cs

```diff
@@ -114,6 +114,7 @@ internal class InitializeHandler : IRequestHandler<InitializeParams, InitializeR
                 _initializeResult.Capabilities.SignatureHelpProvider = null;
                 _initializeResult.Capabilities.ImplementationProvider = false;
 
+                ((VSInternalServerCapabilities)_initializeResult.Capabilities).SupportsDiagnosticRequests = false;
```

Comment:
This should be in your other PR.. if this merges without that, we'll break diagnostics.